src/components/IntlTelInputApp.js

```diff
@@ -222,6 +222,9 @@ class IntlTelInputApp extends Component {
   }
 
   setTelRef = (ref) => {
+    if (typeof this.props.telInputRefCallback === 'function') {
+      this.props.telInputRefCallback(ref);
+    }
     this.tel = ref;
```

Comment:
There is `this.tel`, this wasn't enough?

Reply:
I imagine that the answer is no :)

Reply:
good catch. it actually allows to get the number input ref from IntlTelInputApp, when just passing a ref to it. so the extra prop is not required

Reply:
removed the commit